src/components/app/ListOfPublishedProfiles.css

```diff
@@ -2,14 +2,67 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/*
+ * This implements the layout of the list of recorded profiles. This is used in
+ * 2 locations: in the full uploaded recordings page, or as a small widget in
+ * the home page.
+ *
+ * This is a list, where each list item's layout is:
+ *
+ * /-----------------------------------------------------------------------------------------------------\
+ * | li.publishedProfilesListItem - This is one line in the list.                                        |
+ * | /-------------------------------------------------------------\ /---------------------------------\ |
+ * | | a.publishedProfilesLink                                     | | .publishedProfilesActionButtons | |
+ * | | /------------------------\ /------------------------------\ | |                                 | |
+ * | | | .publishedProfilesDate | | .publishedProfilesInfo       | | | (This element is optional.)     | |
+ * | | |                        | | /--------------------------\ | | |                                 | |
+ * | | |                        | | | .publishedProfilesName   | | | |  /---------------\              | |
+ * | | |                        | | \--------------------------/ | | |  | delete button |              | |
+ * | | |                        | | /--------------------------\ | | |  \---------------/              | |
+ * | | |                        | | | profile metainfo summary | | | |                                 | |
+ * | | |                        | | \--------------------------/ | | | Eventually we'll have more      | |
+ * | | \------------------------/ \------------------------------/ | | buttons here.                   | |
+ * | \-------------------------------------------------------------/ \---------------------------------/ |
+ * \-----------------------------------------------------------------------------------------------------/
+ *
+ * We use nested Flex boxes to achieve this layout.
+ */
+
 .publishedProfilesList {
   padding: 0;
   margin: 0;
   list-style: none;
 }
 
+.publishedProfilesListItem {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+}
+
+.publishedProfilesLink {
+  display: flex;
+  min-width: 0;
+  flex: auto;
+  padding: 4px 8px; /* 4px as vertical padding so that it's 8 when added up, and 8px for the left and right padddings */
+  color: inherit;
+  text-decoration: none;
+}
+
+.publishedProfilesListItem_ConfirmDialogIsOpen .publishedProfilesLink,
+.publishedProfilesListItem:hover .publishedProfilesLink,
+.publishedProfilesListItem:focus-within .publishedProfilesLink {
+  background: var(--grey-90-a10);
```

Comment:
Previously, this was `var(--grey-90-a05)`, but I thought this wasn't visible enough... I remember we discussed it in the past, so I thought I'd highlight the change.